### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by Yuliia-Balandiuk**

### CODE REVIEW

File: javascript/javascript1/week1/age-ify.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
```
Comment (cecastosic) [inline javascript/javascript1/week1/age-ify.js]: why let?

File: javascript/javascript1/week1/age-ify.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+console.log(`You will be ${age} years old in ${yearFuture}.`);
```
Comment (cecastosic) [inline javascript/javascript1/week1/age-ify.js:6]: Nice with template literals 👏

File: javascript/javascript1/week1/ez-namey.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  'front',
+  'way',
+  'company',
+  'department',
+];
+
+const randomNumberFirstWords = Math.floor(Math.random() * 10);
+const randomNumberSecondWords = Math.floor(Math.random() * 10);
+
+let startupName = `${firstWords[randomNumberFirstWords]} ${secondWords[randomNumberSecondWords]}`;
```
Comment (cecastosic) [inline javascript/javascript1/week1/ez-namey.js]: why let?

File: javascript/javascript1/week1/goodboy-oldboy.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
```
Comment (cecastosic) [inline javascript/javascript1/week1/goodboy-oldboy.js]: why let?

File: javascript/javascript1/week1/goodboy-oldboy.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+let dogYear = dogYearFuture - dogYearOfBirth;
+let creature = 'human';
```
Comment (cecastosic) [inline javascript/javascript1/week1/goodboy-oldboy.js:6]: interesting variable name 🙂

File: javascript/javascript1/week1/housey-pricey.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+const wide = 5;
```
Comment (cecastosic) [inline javascript/javascript1/week1/housey-pricey.js]: You need to do the calculation for both houses, Peter's and Julia's. I would suggest to put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]`  `const gardenSizeInM2 = [100,70]`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?

File: javascript/javascript1/week1/housey-pricey.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+const deep = 11;
+const high = 8;
+const gardenSizeInM2 = 70;
+const houseCosts = 1000000;
+
+let volumeInMeters = wide * deep * high;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let amount = 'much';
+
+if (houseCosts < housePrice) {
```
Comment (cecastosic) [inline javascript/javascript1/week1/housey-pricey.js]: what if `houseCosts` higher or equal in comparison with `housePrice`?

File: javascript/javascript1/week1/index.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (cecastosic) [inline javascript/javascript1/week1/index.html]: I am sure you can find a better title 🙂

File: javascript/javascript1/week1/ez-namey.js [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+  'team',
+  'track',
+  'area',
+  'front',
+  'way',
+  'company',
+  'department',
+];
+
+const randomNumberFirstWords = Math.floor(Math.random() * 10);
```
Comment (cecastosic) [inline javascript/javascript1/week1/ez-namey.js]: maybe `firstRandomNumber`

File: javascript/javascript1/week1/ez-namey.js [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+const randomNumberSecondWords = Math.floor(Math.random() * 10);
```
Comment (cecastosic) [inline javascript/javascript1/week1/ez-namey.js]: and `secondRandomNumber`

**Review by cecastosic — COMMENTED** [review]
Good job @Yuliia-Balandiuk 💪 Please be careful when you should use `let` and when `const`. Maybe this video can help https://youtu.be/1mgLWu69ijU I also added some ideas on how to refactor your code. 🌸 Please use Prettier for formatting.

File: javascript/javascript1/week1/housey-pricey.js (hunk 6 above)
Comment (Yuliia-Balandiuk) [reply javascript/javascript1/week1/housey-pricey.js]: there is no less code. but it has become more universal

**Pull request merged**